346/Français.txt

```diff
@@ -1,13 +1,13 @@
-BRIAR&#160;: Quel est le diagnostic ? Aura-t-elle des atteintes persistantes?
+<p>BRIAR&#160;: Quel est le diagnostic ? Aura-t-elle des s&eacute;quelles&#160;?</p>
```

Comment:
For the record, <p> and <br> tags shouldn't be needed. The website will parse these files as markdown, and is set to respect single linebreaks.

In the same vein, you can replace <em> and <strong> tags with `*` and `**` respectively. You can also double them up as `***` which will mark text as both italic and bolded.

Reply:
Okay, so markdown for formatting and Good Ol' HTML for accents and ASCII miscellanea.
Will do!

Reply:
Yup, that should work fine! Please let me know if you run into any edge cases where there are problems, as that will be good to know for the long run.